### PR TITLE
BERT Functionality

### PR DESCRIPTION
**Pull request by jvargas2**
Includes model, pipeline, PyTorch datasets, a new text extractor pipeline component, and new CLI arguments for changing learning rate and epochs. Also, fixes a Windows encoding bug and improves commenting for BiLSTM functionality.

### CODE REVIEW

**Comment by swfarnsworth** [comment]
Thank you for implementing BERT! I fixed some compatibility issues and will put in a new PR.
